### PR TITLE
React Capstone

### PR DESCRIPTION
**Pull request by HabibPayenda**
### **The following requirements were implemented in the project:**

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.
- On the details page, the web app retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.

Please check if I have missed something.

**Thank you.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,60 @@
...
+- React
+- Redux
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://astonishing-piroshki-2b21e0.netlify.app/)
 
 
 ## Getting Started
```
Comment (Deepakdanger) [inline README.md]: - [x] There should be a video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,60 @@
...
+
 ### Deployment
 
 
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
```
Comment (Deepakdanger) [inline README.md:45]: - [x] The README contains an "Acknowledgements" section that should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/screens/HomeScreen/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    totalPopulation += JSON.parse(countries[i].population);
+  }
+  return (
+    <div className="homeScreen">
+      <Navbar />
+      <Header population={totalPopulation} />
+      <List data={countries} />
+    </div>
+  );
+}
```
Comment (Deepakdanger) [inline src/screens/HomeScreen/index.js:26]: - [x] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @HabibPayenda, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required changes
_Check the comments under the review._


## Optional changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/screens/HomeScreen/index.js (hunk 3 above)
Comment (HabibPayenda) [reply src/screens/HomeScreen/index.js:26]: Hello @Deepakdanger, 
**Hope everything is going great by your side**

These are the project requirements that we were given for the home page: 

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.

HomePage

**As you can see in the screenshot from the project we can click on a category which is a card then we are taken to the details screen where the app will show details about that page.**

Kindly check this requirement once more.

**Review by OLIPLICHE — APPROVED** [review]
Hi @HabibPayenda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the required change.👏
- The App and Tests are perfect 👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
# Optional: 
- **From the general requirement**: Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Please try your best not to forget `Gitflow` or `Github flow` and not to be confused, because next time your project will be invalidated
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**